ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -41,6 +45,13 @@ build_root:
     namespace: openshift
     tag: golang-1.16
 images:
+- dockerfile_path: Dockerfile.assisted-service-build
+  from: coreos_centos_centos8
+  inputs:
+    openshift_release_golang-1.16:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.16
```

Comment:
Wasn't this changed to be the postgres image?

Reply:
Depends on https://github.com/openshift/release/pull/24123